packages/ui/src/components/AppContent/index.tsx

```diff
@@ -11,10 +14,26 @@ export type Props = {
 };
 
 const AppContent = ({ children, theme }: Props) => {
+  const [loading, setLoading] = useState(false);
+  const [toast, setToast] = useState('');
+
+  const context = useMemo(() => ({ toast, loading, setLoading, setToast }), [loading, toast]);
+
+  // Prevent internal eventListener rebind
+  const hideToast = useCallback(() => {
+    setToast('');
+  }, []);
+
   return (
-    <div className={classNames(styles.content, styles.universal, styles.mobile, styles[theme])}>
-      {children}
-    </div>
+    <PageContext.Provider value={context}>
+      <div
+        id="main"
```

Comment:
why not using a `<main>` tag?

Reply:
good call, BTW it will be used as querySelector for react-modal root element finding.